app/controllers/concerns/listings_toolkit.rb

```diff
@@ -200,10 +199,12 @@ def listing_updatable?
   end
 
   def at_least_one_param_present?
-    MANDATORY_FIELDS_FOR_UPDATE.any? { |i| listing_params.include? i }
+    MANDATORY_FIELDS_FOR_UPDATE.any? { |field| listing_params.include?(field) }
```

Comment:
I'm a bit torn on this one. While variable names matter, I do believe that they are also a function of scope, so the narrower the scope, the shorter the name can be. Now while `i` certainly wasn't a good choice I think `f` would be perfectly fine but this is obviously ok too.